runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -114,6 +141,17 @@ public MotherlodeSession getSession()
 		return session;
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick event)
+	{
+		if (isInMLM())
+		{
+			CUR_SACK_SIZE = client.getSetting(Varbits.SACK_NUMBER);
```

Comment:
Non-constants should not be named with all caps. Also there is a varbit changed event maybe that would be better for this.